concordia/admin/__init__.py

```diff
@@ -233,6 +241,53 @@ def get_urls(self):
 
         return custom_urls + urls
 
+    def save_model(self, request, obj, form, change):
+
+        old_campaign = Project.objects.get(pk=obj.pk).campaign.slug
+        old_project = Project.objects.get(pk=obj.pk).slug
+
+        super().save_model(request, obj, form, change)
+        # call boto3 api to move project files from older campaign to newer campaign
+        # call boto3 api to update project name within same campaigns and transfer files
+        try:
+
+            new_campaign = obj.campaign.slug
+            new_project = obj.slug
+            # aws_profile = "AWSCrowdlocgov"
+            bucket_name = getattr(settings, "S3_BUCKET_NAME", None)
+            session = boto3.session.Session()
+            s3_resource = session.resource("s3")
+            top_level_bucket = s3_resource.Bucket(bucket_name)
+            filter_from = old_campaign + "/" + old_project
+            if old_campaign != new_campaign or old_project != new_project:
+
+                for obj in top_level_bucket.objects.filter(
+                    Prefix=filter_from + "/"
+                ).all():
+                    # print(obj.key.replace(old_campaign,new_campaign).replace(old_project,new_project))
```

Comment:
Either convert this to a logging debug call or remove it

Reply:
It was left over, removed it.